internal/k8s/controllers/http_route_controller.go

```diff
@@ -28,7 +28,7 @@ type HTTPRouteReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (r *HTTPRouteReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := r.Log.With("route", req.NamespacedName)
+	logger := r.Log.With("http-route", req.NamespacedName)
```

Comment:
Maybe we should just make this `route-name` since the type (http vs tcp) is already in the logger name